### ISSUE TITLE
Inter GPU Usage

### ISSUE THREAD

**Issue by fanxiaochen456** [openvinotoolkit/openvino#12060]
Hello, I use OpenVINO on Windows platform, version 2020 and version 4, and use C++ for reasoning. The model of Inter Graphics card is Inter(R) UHD Graphics 630, and I use Inter GPU for reasoning process discovery. The CPU usage is higher than the GPU usage. How can I improve the GPU usage to improve the inference rate?

**Comment by Iffa-Meah** [comment]
@fanxiaochen456 to improve the inferencing rate, it is recommended to implement model optimization. This would improve final model performance by applying special optimization methods like quantization, pruning, preprocessing optimization, etc. 

You may use the[ OpenVINO Post-Training Optimization Tool](https://docs.openvino.ai/latest/pot_introduction.html#doxid-pot-introduction).

**Comment by brmarkus** [comment]
Can you provide more details about your environment, SW and HW, please? Do you mean OpenVINO in version 2020.4?
(in general it is recommended to use the latest versions like 2021.4.2 or 2022.1).

Can you provide more details about the model(s) you are using?

Can you provide the output of using "benchmark_app" for your CPU and for your GPU?
Would it be possible for your application to make use of batching (parameter "-b"), to make use of parallel streams (parameter "-nstreams"), to trigger multiple inference-requests (parameter "-ireq")?

Do you use (real-time)camera streams (where you are limited by its framerate) or pictures (where you could process many pictures in parallel)?

With (newer) OpenVINO versions you might benefit from the "MULTI" plugin, where you could combine your CPU and your GPU like "-d MULTI:CPU,GPU" or "-d MULTI:GPU,CPU".
Can you try using "MULTI" with "benchmark_app" and also provide the output?

**Comment by Iffa-Meah** [comment]
Closing issue, feel free to re-open or start a new issue if additional assistance is needed.
